virttest/virt_vm.py

```diff
@@ -985,6 +985,7 @@ def verify_dmesg(self, dmesg_log_file=None, connect_uri=None):
         """
         level = self.params.get("guest_dmesg_level", 3)
         ignore_result = self.params.get("guest_dmesg_ignore", "no") == "yes"
+        guest_dmesg_white_list = self.params.get("guest_dmesg_white_list").split(';')
```

Comment:
If there is no "guest_dmesg_white_list" in params, this line will get error like -- AttributeError:  'NoneType' object has no attribute 'split'.